Gruntfile.js

```diff
@@ -16,10 +16,10 @@ module.exports = function (grunt) {
         // Project package file destination.
         pkg: grunt.file.readJSON("package.json"),
         eslint: {
-            all: ["server/src/**/*.js", "ui/src/**/*.js", "tests/**/*.js", "*.js"]
+            all: ["src/**/*.js", "tests/**/*.js", "*.js"]
         },
         jsonlint: {
-            all: ["package.json", ".eslintrc.json", "server/src/**/*.json", "ui/src/**/*.json", "tests/**/*.json"]
+            all: ["package.json", ".eslintrc.json", "themes/**/*.json", "tests/**/*.json"]
```

Comment:
Json5 files such as "sjrk.storyTelling.server.config.json5" and those in the "tests/" directory should be linted too.

Reply:
jsonlint does not currently support json5 format, nor am I able able to find a viable eslint plugin for json5. That said, if I remove the comments from the json5 files, they should pass jsonlint, but if any developer decides to use any of the extra features of json5 they will fail the linting checks.

Reply:
The universal repo has [an example](https://github.com/GPII/universal/blob/master/Gruntfile.js#L23) of linting json5 files.

Reply:
The relevant module is https://github.com/GPII/gpii-grunt-lint-all , written by @the-t-in-rtf which is well worth using